CHANGELOG-0.10.md

```diff
@@ -46,6 +46,7 @@
   of some sort set in place ([#380][])
 * Shipper now exposes metrics that measure time from Application 
   creation/modification to Chart Installation ([#383][])
+* ping pods when moving back in the strategy([#387][]))
```

Comment:
🤔  I am a bit confused about this, maybe we could make a bit clearer. WDYT?

Reply:
oops, copy pasta 🤭